docs/release-notes/8.11.asciidoc

```diff
@@ -1,6 +1,23 @@
 [[release-notes-header-8.11.0]]
 == 8.11
 
+[discrete]
+[[release-notes-8.11.2]]
+=== 8.11.2
+
+[discrete]
+[[enhancements-8.11.2]]
+==== Enhancements
+* Correct References under 'Useful Links' in Risk Score Enablement page ({pull}171089[#171089]).
+
+[discrete]
+[[bug-fixes-8.11.2]]
+==== Bug fixes
+* Fixes a bug that caused the Alerts page to crash if you reloaded it while the preview panel in the alert details flyout was open ({pull}172323[#172323]).
+* Fixes the event analyzer panel width ({pull}172026[#172026]).
+* Makes MITRE ATT&CK® sub-technique cells consider page filters when displaying rules ({pull}170988[#170988]).
+* Fixes a bug with the **Investigate in timeline** action that caused {esql} queries to open in the KQL query bar within Timeline ({pull}170542[#170542]).
```

Comment:
nit: not immediately clear this is the Assistant `Investigate in timeline` action vs Alert investigate in timeline action, but other than that LGTM 👍